### DISCUSSION TITLE
Access to metadata from Prisma Client extension

### DISCUSSION THREAD

**Discussion by berezhinskiyo** [prisma/prisma#23678] [Q&A] [answered]
### Question

Hello,
I used to work with object `dmmf` from `import { Prisma } from '@prisma/client'` to access to models metadata (`Prisma.dmmf.datamodel.models`), it worked well till version 5.8.0, but in latest version 5.11.0 `dmmf` disappear. Where I can find it, or something else to access to models metadata?

### Expected behavior (optional)

It(dmmf) appears in `import { Prisma } from '@prisma/client'`

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment, marked as answer]
Hi @berezhinskiyo 👋 

I just tested with Prisma Client `"@prisma/client": "^5.11.0"` and `"prisma": "^5.11.0". You can still access the `dmmf`as a property on Prisma.
```js
import { Prisma, PrismaClient,  } from '@prisma/client';

 console.log(Prisma.dmmf.datamodel.models);
```

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by berezhinskiyo** [reply]
Thank you!

**Reply by ludralph** [reply]
You're welcome.
